generic/tests/cfg/timedrift.cfg

```diff
@@ -77,25 +64,31 @@
             test_duration = 600
             smp = 2
             stop_time = 60
-            drift_threshold = 10
         - with_signal_stop:
-            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-            # and the time drift is seen because "stop" monitor command also stops PM timer,
-            # so this is unsupported scenario from Win2012.
-            Windows:
-                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
             no RHEL.3.9, RHEL.4
             type = timedrift_with_stop
             stop_interations = 1
             stop_with_signal = yes
             stop_time = 60
-            drift_threshold = 10
-            drift_threshold_single = 3
-            variants:
-                - clock_host:
-                    rtc_clock = host
-                - clock_vm:
-                    rtc_clock = vm
+    variants:
+        - clock_host:
+            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
+            # and the time drift is seen because "stop" monitor command also stops PM timer,
+            # so this is unsupported scenario from Win2012.
+            Windows:
+                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
+            rtc_clock = host
+        - clock_vm:
+            rtc_clock = vm
```

Comment:
`rtc_clock = vm` will be replaced by `rtc_clock = host`  in staf, please update staf accodring. thanks !

Reply:
@yanan-fu thanks.

Yes. A patch has been sent to move "rtc_clock = host" from staf internal cfg to timedrift.cfg.
http://patchwork.virt.bos.redhat.com/patch/75820

Reply:
@yanan-fu thanks.

A staf patch 76349 has been sent. The patch is to move "rtc_clock = host" from internal.cfg to timedrift.cfg.